.github/CONTRIBUTING.md

```diff
@@ -126,53 +129,67 @@ If you want to test a branch before releasing it, you can push up a _built branc
    ```
 
 ## Git workflow
+### Commit style
 
-### Branches
-
-The only "special" branch is the `master` branch which represents the latest stable version of the library. Most PRs with bug fixes, features, and even breaking changes should probably be made against, and ultimately merged into, `master`. The only exception is if you're unsure of changes and need to validate your ideas in production before making an official stable Maker release, which would be most common in the case of breaking changes and major version releases, so let's go over an example of how we'd do one.
+This repository uses [Conventional Commits](https://www.conventionalcommits.org) for simple yet meaningful commit messages. Not only are they user-friendly, they are also machine-readable for automated release notes and versioning.
 
-Let's say Maker is on version `1.5.2` and you have a breaking change, or maybe several, that you'd like to make and that would require a major version release of `2.0.0`. Unless you're very confident in your changes you'll likely want to do a few _pre-releases_ first, and hold out on directly merging your changes directly into `master`. So what you'd do is:
```

Comment:
I don't mind you removing these instructions but I think it's important we offer some guidance on how to do pre-releases. It's kind of uncommon to do pre-releases so I'd like an easy-to-follow step-by-step guide in the docs telling people how to do it.

Reply:
I added a reference to the [semantic-release docs](https://github.com/semantic-release/semantic-release/blob/master/docs/recipes/pre-releases.md#publishing-pre-releases) for this as they explain it very elaborately.